Library/Homebrew/cask/cask.rb

```diff
@@ -330,6 +330,12 @@ def to_h
         "conflicts_with"       => conflicts_with,
         "container"            => container&.pairs,
         "auto_updates"         => auto_updates,
+        "deprecated"           => deprecated?,
+        "deprecation_date"     => deprecation_date,
+        "deprecation_reason"   => deprecation_reason,
+        "disabled"             => disabled?,
+        "disable_date"         => disable_date,
+        "disable_reason"       => disable_reason,
```

Comment:
It looks like `disabled` and `deprecated` are really only necessary on the formula side to aid with HTML generation for `formulae.brew.sh`. Probably be a good idea to add a follow-up to enable that for casks too.

- Example disabled formula: https://formulae.brew.sh/formula/aacgain

Reply:
If we want to slim this, we could easily make the logic on the formule.brew.sh side be `deprecation_date >= today`.